### PR TITLE
stable/prometheus-mongodb-exporter: version bump percona mongdb exporter

### PR DESCRIPTION
**Pull request by igaskin**
also adding extra arg

### CODE REVIEW

**Comment by zanhsieh** [comment]
@igaskin 
Would you mind to bump the chart version to latest please? Thank you.

**Comment by igaskin** [comment]
> @igaskin
> Would you mind to bump the chart version to latest please? Thank you.

Sure thing, just bumped the chart a minor version

**Comment by cpanato** [comment]
@igaskin can you please sign the dco as well

File: stable/prometheus-mongodb-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "v0.7.0"
+appVersion: "v0.10.0"
```
Comment (steven-sheehy) [inline stable/prometheus-mongodb-exporter/Chart.yaml:2]: Bump version in README and sign off your commits.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Comment by zanhsieh** [comment]
@igaskin 
I think you need to set up your git environment correctly. Git seems not be able to differ "iGaskin" and "Issac Gaskin" in your local git repo.

```
Commit sha: df1f1e1, Author: iGaskin, Committer: iGaskin; Expected "iGaskin gask0032@gmail.com", but got "Isaac Gaskin gask0032@gmail.com".
```

**Comment by igaskin** [comment]
> @igaskin
> I think you need to set up your git environment correctly. Git seems not be able to differ "iGaskin" and "Issac Gaskin" in your local git repo.
> 
> ```
> Commit sha: df1f1e1, Author: iGaskin, Committer: iGaskin; Expected "iGaskin gask0032@gmail.com", but got "Isaac Gaskin gask0032@gmail.com".
> ```

Okay, let me take a peak and try to fix the signature.

**Review by steven-sheehy — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**